libs/navigation/driver/in-memory/src/navigation-driver.module.ts

```diff
@@ -20,8 +19,7 @@ export class DaffNavigationInMemoryDriverModule {
     return {
       ngModule: DaffNavigationInMemoryDriverModule,
       providers: [
-        provideDaffNavigationDriver(DaffInMemoryNavigationService),
-        provideDaffInMemoryBackends(DaffInMemoryBackendNavigationService),
+        provideDaffNavigationInMemoryDriver(),
```

Comment:
will this be properly spread? I think you might have too many nested arrays here, I don't know how angular would handle
```ts
providers: [[[provider]]]
```

Reply:
e78273f7d3f6aab54738e2bb4b685a7da7c51edf

Reply:
okay as long as you've verified correct behavior. It's possible that Angular now unwraps deeply nested arrays. I believe at one point they would only go one layer deep